parmed/openmm/parameters.py

```diff
@@ -149,7 +149,13 @@ def _remediate_residue_template(cls, params, residue):
         for lonepair in residue.lonepairs:
             lp_atom = lonepair[1]
             params.atom_types[types[lp_atom]].atomic_number = 0
-            atom.atomic_number = 0
+            residue[lp_atom].atomic_number = 0
+
+        # CHARMM Drude force field includes bonds to lone pairs.  Delete them.
+        for bond in list(residue.bonds):
```

Comment:
You shouldn't need to coerce to a list here -- just iterate over the bonds directly.

Reply:
This is needed for a different reason.  If you modify the list of bonds at the same time you're iterating over it, things get confused and some bonds get missed.  The call to `list()` means we're iterating over an unchanging copy.

Reply:
Oh of course. I’ve done this before. Can you add a comment to this effect?

Reply:
Sure.  (This isn't my code though.  It was already there.  I just moved it to a different point in the method.)